server/routes/private.js

```diff
@@ -50,6 +51,34 @@ router.post('/api/book-appointment', async (req, res) => {
 
 });
 
+router.post('/api/add-recurring-schedule', async (req, res) => {
+    // Validate appropriate parameters are passed into add recurring schedule. 
+    const paramSchema = Joi.object({
+        slotId: Joi.number().integer().required(), //The worker must specify the slot(s) they are available 
```

Comment:
They are only specifying one slotId here based on the Joi.number().integer() rather than an array so I would recommend removing (s) in the comment to avoid future confusion.

Reply:
Thanks @catai9 completed!